src/pages/page-2.js

```diff
@@ -3,13 +3,160 @@ import { Link } from "gatsby"
 
 import Layout from "../components/layout"
 import Seo from "../components/seo"
+import imgLogo from "../images/Luma.png"
+import imgCart from "../images/Cart.png"
 
 const SecondPage = () => (
   <Layout>
     <Seo title="Page two" />
-    <h1>Hi from the second page</h1>
-    <p>Welcome to page 2</p>
+    {/* <h1>Hi from the second page</h1>
+    <p>Welcome to page 2</p> */}
     <Link to="/">Go back to the homepage</Link>
+    <h1></h1>
+
+    <div className="flex flex-row">
+      <div className="divHeaderCell w-8/12">
+        <img src={imgLogo} />
+      </div>
+      <div className="divHeaderCell w-3/12">
+        <input type="text" placeholder="Search entire store here..." />
+      </div>
+      <div className="divHeaderCell w-1/12">
+        <img src={imgCart} />
+      </div>
+    </div>
+
+    <div className="flex py-0 font-bold bg-gray-200 invisible md:px-2 md:visible md:py-2">
+      <label>Category</label>
+    </div>
+    <div className="invisible md:visible md:py-3"></div>
+    <label className="text-2xl sm:text-3xl">Request For Quote</label>
+    <div className="py-4"></div>
+
+    <div className="flex flex-col lg:flex-row lg:space-x-4">
+      {/* Contact Information */}
+      <div className="flex flex-col lg:w-1/3 lg:flex-col">
+        <div>
+          <label className="sectionHeader">Contact Information</label>
+          <hr className="sectionLine"></hr>
+        </div>
+        <div className="flex flex-col md:flex-row lg:flex-col">
+          <div className="divCell">
+            <input type="text" id="firstName" placeholder="First name*" />
+          </div>
+          <div className="divCell">
+            <input type="text" id="lastName" placeholder="Last name*" />
+          </div>
+        </div>
+        <div className="flex flex-col md:flex-row lg:flex-col">
+          <div className="divCell">
+            <input type="text" id="email" placeholder="Email*" />
+          </div>
+          <div className="divCell">
+            <input type="text" id="phone" placeholder="Phone" />
+          </div>
+        </div>
+      </div>
+
+      {/* Address */}
+      <div className="sectionSecondBlock flex flex-col lg:w-2/3 lg:flex-col">
+        <div>
+          <label className="sectionHeader">Address</label>
+          <hr className="sectionLine"></hr>
+        </div>
+        <div className="divCell">
+          <input type="text" id="streetLine1" placeholder="Street line 1*" />
+        </div>
+        <div className="divCell">
+          <input type="text" id="streetLine2" placeholder="Street line 2" />
+        </div>
+
```

Comment:
get rid of open space.

Reply:
Painful to remove all these... but ok... for now :)